variables.tf

```diff
@@ -426,6 +426,12 @@ variable "kafka_topic_name" {
   default     = ""
 }
 
+variable "s3_bucket_expiry" {
```

Comment:
```suggestion
variable "s3_bucket_expiry_days" {
```
Is it possible to rename this so it's more descriptive?

Reply:
I would suggest `s3_object_expiration_days`

Reply:
That's even better, yes.